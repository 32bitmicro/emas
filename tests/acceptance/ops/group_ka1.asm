AWT r0, 0
TRB r0, 0
IRB r0, 0
DRB r0, 0
CWT r0, 0
LWT r0, 0
LWS r0, 0
RWS r0, 0
