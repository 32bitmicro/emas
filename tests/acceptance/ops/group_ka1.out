@ 0x0000 : 0xc000  /  110 000 0 000 000 000  /  49152
@ 0x0001 : 0xc400  /  110 001 0 000 000 000  /  50176
@ 0x0002 : 0xc800  /  110 010 0 000 000 000  /  51200
@ 0x0003 : 0xcc00  /  110 011 0 000 000 000  /  52224
@ 0x0004 : 0xd000  /  110 100 0 000 000 000  /  53248
@ 0x0005 : 0xd400  /  110 101 0 000 000 000  /  54272
@ 0x0006 : 0xd800  /  110 110 0 000 000 000  /  55296
@ 0x0007 : 0xdc00  /  110 111 0 000 000 000  /  56320
